src/scene.rs

```diff
@@ -203,12 +234,19 @@ impl<I: ImageSize> Scene<I> {
         };
 
         if removed.is_some() {
+            self.dead_sprites.remove(&id);
             self.stop_all_including_children(removed.as_ref().unwrap());
         }
 
         removed
     }
 
+    /// Remove the child by `id` from the scene's children or grandchild once
+    /// all of its animations have finished.
+    pub fn remove_child_when_done(&mut self, id: Uuid) {
```

Comment:
ah there's a bug here - if no animations are present there will never be an update step so sprite will never be removed. Need to fix code and update docs. (Will do this later in the week, getting too late now.)